lib/omniauth/strategies/azure_oauth.rb

```diff
@@ -19,14 +19,10 @@ class AzureOauth < OmniAuth::Strategies::OAuth2
       end
 
       info do
-        raw_info['groups'] = graph_groups if ENV['AZURE_GRAPH_GROUPS'].present?
+        raw_info['groups'] = graph_groups
```

Comment:
raw_info['groups'] should have in it the groups claim, if we always call graph_groups on it we'll always return an empty array which probably isn't what we want. I took a slightly different approach on Friday. https://github.com/psu-stewardship/scholarsphere/pull/840/files

Reply:
Right, I saw that. What isn't clear is where or how the groups key is getting populated prior to this, assuming it is...